Jenkinsfile.sdk

```diff
@@ -0,0 +1,47 @@
+@Library('lisk-jenkins') _
+
+pipeline {
+	agent { node { label 'lisk-commander' } }
```

Comment:
`lisk-sdk`?

Reply:
This is just a label be we could change it.